frontend/src/components/application/AppVendingControls/OwnershipTokens/TokenList.tsx

```diff
@@ -18,11 +23,24 @@ interface Props {
 const TokenList: FunctionComponent<Props> = ({ app }) => {
   const { t } = useTranslation()
 
-  const {
-    value: tokens,
-    status,
-    error,
-  } = useAsync(useCallback(() => getVendingTokens(app.id), [app.id]))
+  const [tokens, setTokens] = useState(null)
+  const [status, setStatus] = useState("pending")
+  const [error, setError] = useState(null)
+
+  const doFetch = useCallback(async () => {
+    try {
+      const fetch = await getVendingTokens(app.id)
+      setTokens(fetch)
+      setStatus("done")
```

Comment:
The old implementation used `success`, not sure if that matters.

```suggestion
      setStatus("success")
```